spec/src/main/asciidoc/core/events.asciidoc

```diff
@@ -166,7 +166,13 @@ The methods `fire()` and `fireAsync()` fire an event with the specified qualifie
 If the container is unable to resolve the parameterized type of the event object, it uses the specified type to infer the parameterized type of the event types.
 
 The method `fireAsync()` may be called with a specific `Executor` object to be used to invoke observers method.
-The container should provide a default `Executor` when `fireAsync()` is called without specifying one.
+
+The following elements are container specific:
+
+* the default `Executor` used by the container when `fireAsync()` is called without specifying an `Executor`,
+* the `CompletionStage` returned by `fireAsync` methods,
+* all dependent stages of this initial CompletionStage, and
+* the default asynchronous execution facility used for this stages.
```

Comment:
This last point seems to me unnecesarry IMO. It just repeats CompletionStage javadoc.